lib/rules/media-type-no-deprecated/__tests__/index.mjs

```diff
@@ -0,0 +1,188 @@
+import rule from '../index.mjs';
+
+const { messages, ruleName } = rule;
+
+testRule({
+	ruleName,
+	config: [true],
+
+	accept: [
+		{
+			code: '@media all {}',
+		},
+		{
+			code: '@media screen {}',
+		},
+		{
+			code: '@media print {}',
+		},
+		{
+			code: '@media {}',
+		},
+		{
+			code: '@media screen and (min-width: 600px) {}',
+		},
+		{
+			code: '@media all and (color: 8) {} ',
+		},
+		{
+			code: '@media screen and (min-width: 600px), print and (max-width: 800px) {}',
+		},
+	],
+
+	reject: [
```

Comment:
Some reject cases don't have endLines, can you please add them?

Reply:
Added